src/Apps/Order2/Routes/Checkout/Components/FulfillmentDetailsStep/Order2PickupForm.tsx

```diff
@@ -45,8 +45,8 @@ export const Order2PickupForm: React.FC<Order2PickupFormProps> = ({
     option => option.type === "PICKUP",
   )
 
-  const setOrderPickupDetails = useSetOrderPickupDetailsMutation()
-  const setOrderFulfillmentOption = useSetOrderFulfillmentOptionMutation()
+  const setOrderPickupDetails = useOrder2SetOrderPickupDetailsMutation()
+  const setOrderFulfillmentOption = useOrder2SetOrderFulfillmentOptionMutation()
```

Comment:
most of the rest of the changes in this file are moving towards consistent naming conventions for all Order2 mutations.